Healthy/Classes/Extensions/UIButton+Style.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  UIButton+Style.swift
+//  Healthy
+//
+//  Created by Ahmed Nasr on 13/05/2023.
+//
+
+import UIKit
+
+
+//MARK: - button style -
+//
+extension UIButton {
+    enum ButtonStyle {
+        case primary
+        case secondary
+        case disable
+    }
+}
+
+
+//MARK: - apply button style -
+//
+extension UIButton {
+    func applyButtonStyle(style: ButtonStyle) {
+        buttonStyleAttributes(style: style)
+    }
+}
+
+
+//MARK: - button style attribute -
+//
+extension UIButton {
+    private func buttonStyleAttributes(style: ButtonStyle) {
```

Comment:
This is a noun, I do expect returned attributes when I first read the method. Since you're **doing** something, this should start with a verb.